src/rust/organization-management/src/lib.rs

```diff
@@ -9,11 +9,19 @@ pub struct OrganizationManagementServiceConfig {
     #[clap(long, env)]
     pub organization_management_healthcheck_polling_interval_ms: u64,
     #[clap(long, env)]
-    pub organization_management_db_hostname: String,
-    #[clap(long, env)]
-    pub organization_management_db_port: u16,
+    pub organization_management_db_address: String,
     #[clap(long, env)]
     pub organization_management_db_username: String,
     #[clap(long, env)]
-    pub organization_management_db_password: String,
+    pub organization_management_db_password: grapl_config::SecretString,
+}
+
+impl grapl_config::ToPostgresUrl for OrganizationManagementServiceConfig {
```

Comment:
99% chance that i proc macro this before EOD

Reply:
i was thinking that if we are to proc macro for ToPostgresUrl that we could just as well derive the fields in the first place, where we take a prefix.

```
#[derive(grapl::PostgresConfig(prefix = "EXAMPLE_SERVICE")]
```

maybe that's too much marco tho idk